src/services/project/api.ts

```diff
@@ -72,7 +72,7 @@ export async function updateProjectNotes(id: string, notes: string) {
 }
 
 export async function getUserPublicProjects(userId: string) {
-  return await createAdminClient()
+  return await createClient()
```

Comment:
This has to be `createAdminClient()` otherwise for anonymous users it's not gonna fetch the repositories.

But I'd like to know more, did you change it because it didn't work on your machine or other reasons?

Reply:
Indeed, it didn't. The only way I could access the projects was changing it to createClient. I can provide you with proof in the discord or somewhere else if you want

Reply:
That's interesting, than it's something that needs to get fixed, here we must keep the admin client or in the deployed version it won't work.

Reply:
Alright, done